boards/xtensa/esp32/esp32-devkitc/src/esp32_bringup.c

```diff
@@ -223,6 +223,15 @@ int esp32_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_ESP32_WIFI_BT_COEXIST
+  ret = esp32_wifi_bt_coexist_init();
+  if (ret)
+    {
+      syslog(LOG_ERR, "ERROR: Failed to initialize Wi-Fi and BT \
+                       coexistence support\n");
```

Comment:
```suggestion
      syslog(LOG_ERR, "ERROR: Failed to initialize Wi-Fi and BT "
             "coexistence support\n");
```
All those whitespaces from line 231 would be output since they are part of the string.